jaraco/abode/devices/camera.py

```diff
@@ -173,6 +184,29 @@ def snapshot_data_url(self, get_snapshot=True):
 
         return f"data:image/jpeg;base64,{self._snapshot_base64}"
 
+    def start_kvs_stream(self, path):
+        """Start KVS Stream for camera."""
+        KVS_STREAM_URL = urls.CAMERA_INTEGRATIONS + self.id + '/kvs/stream'
+
+        response = self._client.send_request(method="post", path=KVS_STREAM_URL)
+        response_object = response.json()
+
+        log.debug("Camera KVS Stream URL (put): %s", KVS_STREAM_URL)
+        log.debug("Camera KVS Stream Response: REDACTED (due to embedded credentials)")
+
+        if response_object['channelEndpoint'] is None:
+            raise jaraco.abode.Exception(ERROR.START_KVS_STREAM)
+
+        log.info("Started camera %s KVS stream:", self.id)
+
+        if self.stream_details_to_file(response.text, path):
+            log.info(
+                "Saved KVS stream endpoint data to %s for device id %s", path, self.id
+            )
+            return True
+
+        return False
```

Comment:
I try to avoid boolean literals. What do you think about this instead?

```suggestion
        if status := self.stream_details_to_file(response.text, path):
            log.info(
                "Saved KVS stream endpoint data to %s for device id %s", path, self.id
            )

        return status
```